packages/cache/src/types.ts

```diff
@@ -0,0 +1,16 @@
+import type { Tuple } from './utility-types'
+
+export type Key = Tuple
```

Comment:
```suggestion
export type CacheKey = Tuple
```

Reply:
I modified it https://github.com/toss/suspensive/pull/1083/commits/56c46ac2d3d1777303d3049d0dcc776c9cf45f89

Reply:
The reason for exporting an additional Key instead of just CacheKey is to provide a more versatile Key type, similar to the use case of [hashKey](https://github.com/toss/suspensive/pull/1083/files#diff-3f6648bedc66515fc0be95d8cc88038310f5fd2c460dffdb2dce72bf8ce5b068).

Reply:
Why should hashKey use Key, not CacheKey?

Reply:
I chose to use the type 'Key' because I believe the key for hashKey should not be limited to just a cache key.

Reply:
Then I want to know why you believe it. I don't think Key always tuple. Why we need same type Key and CacheKey

Reply:
Since the utility function name is hashKey, I thought Key would be more suitable than CacheKey as it should not be limited to a specific situation. However, upon reconsideration, since hashKey is used only for hashing cacheKey, renaming the function to hashCacheKey as you suggested would clarify its purpose. Therefore, it would be better to change the type from Key to CacheKey.

Reply:
Ah, I understood. Thanks for your explaination